tests/Doctrine/ODM/MongoDB/Tests/Functional/DocumentPersisterTest.php

```diff
@@ -651,7 +651,7 @@ public function testExecuteInsertsRespectsWriteConcern(string $class, $writeConc
 
     /** @psalm-param class-string $class */
     #[DataProvider('dataProviderTestWriteConcern')]
-    public function testExecuteInsertsOmitsWriteConcernInTransaction(string $class): void
+    public function testExecuteInsertsOmitsWriteConcernInTransaction(string $class, $writeConcern): void
```

Comment:
I would suggest a `string|int` type for `$writeConcern` to silence PHPStan:
```suggestion
    public function testExecuteInsertsOmitsWriteConcernInTransaction(string $class, string|int $writeConcern): void
```